pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -78,6 +81,27 @@ def test_unary_ops(self):
 
 
 class TestTimedeltaComparison(object):
+    @pytest.mark.parametrize('tick_cls', [Nano, Micro, Milli, Second,
```

Comment:
dont' we have this in a fixture? tick_classes?

Reply:
just pushed with this change